drivers/clock_control/nrf_power_clock.c

```diff
@@ -280,6 +295,11 @@ static int clock_async_start(struct device *dev,
 				config->start_handler(dev) : true;
 		if (do_start) {
 			DBG(dev, subsys, "Triggering start task");
+
+			if (IS_ENABLED(CONFIG_NRF52_ANOMALY_132_WORKAROUND)) {
+				anomaly_132_workaround(config->start_tsk);
```

Comment:
Passing of the start task to `anomaly_132_workaround` is a bit misleading. IMO it would be clearer to check the task in the `if` right above, as this anomaly applies to LFCLK only.

Reply:
done